modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractPythonCodegen.java

```diff
@@ -691,10 +688,7 @@ public String toApiName(String name) {
 
     @Override
     public String toApiVarName(String name) {
-        if (name.length() == 0) {
-            return "default_api";
```

Comment:
What happens when name length is 0 with your update?

Reply:
after update, it will call method: `toApiName(name)`

```java
    public String toApiName(String name) {
        if (name.length() == 0) {
            return "DefaultApi";
        }
        return camelize(apiNamePrefix + "_" + name + "_" + apiNameSuffix);
    }
```

this method will get default name where name length is 0